stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -107,17 +108,54 @@ private static String removeSuffix(String str, String[] suffixesToRemove) {
     Util.throwIfNull(handler);
     SQLiteDatabase database = openDatabase(databaseName);
     try {
-      Cursor cursor = database.rawQuery(query, null);
-      try {
-        return handler.handleResult(cursor);
-      } finally {
-        cursor.close();
+      String trimmedSql = query.trim();
+      String prefixSql = "";
+      if (trimmedSql.length() >= 3) {
+        prefixSql = trimmedSql.substring(0, 3).toUpperCase();
       }
+      if (prefixSql.equals("UPD") || prefixSql.equals("DEL")) {
```

Comment:
Re `toUpperCase`, I missed this but prefer `equalsIgnoreCase` on the full word.  It's better performing and IMO, clearer.  Also, not applicable in this case but `a.equalsIgnoreCase(b)` is not strictly equal to `a.toUpperCase().equals(b.toUpperCase())`.  Casing is defined very differently in a lot of languages.

These are just nits tho, no big deal.